### PR TITLE
refactor scenariorun commands

### PR DESCRIPTION
**Pull request by Leopold-Cramer**

### CODE REVIEW

File: Babylon/commands/api/scenarioruns/cumulated_logs.py [hunk 1]
```diff
@@ -22,18 +23,13 @@ def cumulated_logs(state: Any, azure_token: str, organization_id: str, scenarior
...
     Get the cumulated logs for the scenarioRun
     """
-    state = state['services']
-    state['api']['organization_id'] = organization_id or state['api']['organization_id']
-    state['api']['scenariorun_id'] = scenariorun_id or state['api'].get('scenariorun_id')
-    if state['api']['scenariorun_id'] is None:
-        logger.error(f"scenariorun : {state['api']['scenariorun_id']} does not exist")
-        return CommandResponse.fail()
-
+    scenariorun_state = state['services']
```
Comment (esasova) [inline Babylon/commands/api/scenarioruns/cumulated_logs.py:26]: elsewhere we use _service_state_, is there a special need to precise _scenariorun_state_ ? :slightly_smiling_face:

File: Babylon/commands/api/scenarioruns/service/api.py [hunk 2]
```diff
@@ -9,22 +9,18 @@
 
 class ScenarioRunService:
 
-    def __init__(self, state: dict, azure_token: str, spec: Optional[dict] = None):
+    def __init__(self, azure_token: str, state: dict, spec: Optional[dict] = None):
```
Comment (esasova) [inline Babylon/commands/api/scenarioruns/service/api.py:12]: finally, this service doesn't need a `spec` argument, could you please remove it ? :slightly_smiling_face:

**Review by esasova — APPROVED** [review]
just two questions but LGTM :+1:

**Review by vysahjk — APPROVED** [review]
👍🏽 +2

**Pull request merged**